app/services/taxes/destroy_service.rb

```diff
@@ -33,5 +34,10 @@ def draft_invoice_ids
         .draft
         .pluck(:id)
     end
+
+    def destroy_applied_taxes
+      billing_entity = tax.organization.default_billing_entity
+      BillingEntities::Taxes::RemoveTaxesService.call(billing_entity:, tax_codes: [tax.code])
```

Comment:
Don't we want to raise/catch potential failures coming from the this service?

Reply:
we kinda shouldn't fail here, because the only error risen from `billing_entities/taxes` services is tax not found, but this is already processed at the beginning of method `call`...
The only other thing that can fail will be failing to save `BillingEntity::AppliedDax` object, where we have no validations anyway

Reply:
Alright! Thank you for the explanation :+1: